components/Landing/Info.tsx

```diff
@@ -7,17 +7,33 @@ import { colors } from '../../stylesheets/colors';
 import Link from 'next/link';
 
 const Grid = styled.div`
-  margin: 100px 0;
+  min-height: 20vh;
+  @media (min-width: 1023px) {
```

Comment:
Maybe we could have the media query or the pixel value as a css-variable or or ts variable such as the colors since it is reused so much?